### ISSUE TITLE
Document the DOM PDF version requirements

### ISSUE THREAD

**Issue by lukeholder** [craftcms/docs#42]
### Description

Craft Commerce requires PHP `7.0` or greater (just like Craft CMS).

The default craft project composer.json sets the platform to `7.0` https://github.com/craftcms/craft/blob/master/composer.json#L36

This platform requirement in the project will favour dependancies that are compatible with `7.0`

In Commerce we require `dompdf/dompdf ^0.8.3` which runs fine on PHP `7.0`

If you run your project on PHP `7.4` and have dompdf `0.8.3` installed, you will get errors.

dompdf `0.8.4+` requires PHP `7.1` but these versions will not be installed due to your projects composer.json platform requirement.

Bottom line, if you are running PHP `7.4` update your projects platform to `7.4` (or `7.1` or greater) and do a composer update and dompdf will be upgraded to `0.8.4+` and work again.

@mattstein please ask me questions if the above is not clear. Probably best in the upgrade & requirements docs pages.

**Comment by mattstein** [comment]
@lukeholder let me know what you think of https://github.com/craftcms/docs/pull/50!

I didn’t include the note in the v3 upgrade notes since the Dompdf version requirement is exactly the same in Commerce 2.
